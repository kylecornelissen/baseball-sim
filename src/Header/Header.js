import React from 'react';
import { Link } from 'react-router-dom';
import './Header.scss';

const Header = () => {
  return (
    <header>
      <img src={process.env.PUBLIC_URL + '/menu.svg'} alt="hamburger-menu-btn"/>
      <Link className="title-header" to='/games'>
        <h1>Just Baseball.<span>Beta</span></h1>
      </Link>
    </header>
  )
}

export default Header;
