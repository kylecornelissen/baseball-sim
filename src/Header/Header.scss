@import '../variables.scss';

header {
  background: $carbon;
  border-bottom: 3px solid $blueaqua;
  position: fixed;
  height: 80px;
  width: 100%;
  top: 0px;
  left: 0px;
  z-index: 500;
  display: flex;
  flex-direction: row;
  justify-content: flex-start;
  align-content: center;
}

.hamburger-menu-logo {
  width: 55px;
  padding: 0% 3%;
}

.title-header {
  color: $blueaqua;
  text-decoration: none;
  width: 25%;
  display: flex;
  flex-direction: column;
  justify-content: center;
}

h1 {
  font-size: 39px;
  margin: 0;
  padding-left: 1%;
  >span {
    color: $white;
    font-size: 1.2vw;
    vertical-align: top;
  }
}
