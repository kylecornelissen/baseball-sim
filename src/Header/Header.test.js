import React from 'react';
import { shallow } from 'enzyme';
import Header from './Header';

describe('Header', () => {
  it('renders the website title in h1 and Beta in span', () => {
    const wrapper = shallow(<Header />);
    const title = <h1>Just Baseball.<span>Beta</span></h1>

    expect(wrapper.contains(title)).toEqual(true);
  });

  it('should match the snapshot with all data passed in correctly', () => {
    const wrapper = shallow(<Header

    />);

    expect(wrapper).toMatchSnapshot();
  });
});
