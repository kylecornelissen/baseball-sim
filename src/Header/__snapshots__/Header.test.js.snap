// Jest Snapshot v1, https://goo.gl/fbAQLP

exports[`Header should match the snapshot with all data passed in correctly 1`] = `
<header>
  <img
    alt="hamburger-menu-btn"
    src="/menu.svg"
  />
  <Link
    className="title-header"
    to="/games"
  >
    <h1>
      Just Baseball.
      <span>
        Beta
      </span>
    </h1>
  </Link>
</header>
`;
